### PR TITLE
Preload fonts

### PR DESCRIPTION
**Pull request by taylor-vann**
Fixes preloads of fonts, should be firefox compatible.

### CODE REVIEW

File: demos/dialog/index.js [hunk 1]
```diff
@@ -34,3 +36,61 @@ window.toggleActions.onclick = function() {
   dialog.hideActions = hideAction;
   hideActionSpan.innerText = hideAction;
 };
+
+class DailogWithFormValidation extends LitElement {
```
Comment (asyncLiz) [inline demos/dialog/index.js:40]: Think this is part of something else?

File: demos/dialog/index.html [hunk 2]
```diff
@@ -153,6 +155,9 @@
         Secondary
       </mwc-button>
     </mwc-dialog>
+    <mwc-button data-num="8" raised>Form Validation</mwc-button>
```
Comment (asyncLiz) [inline demos/dialog/index.html:158]: Ditto here

File: demos/top-app-bar_iframe/index.html [hunk 3]
```diff
@@ -20,14 +20,15 @@
...
+  <link rel="preload" href="https://fonts.googleapis.com/css?family=Material+Icons&display=block" as="style" type="text/css" crossorigin>
   <script type="module" src="index.js"></script>
   <script nomodule src="../../node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
   <style>
     .unresolved main {
       opacity: 0;
     }
-  </style>
-  <link href="../shared/shared.css" rel="stylesheet">
+  </style><link href="../shared/shared.css" rel="stylesheet">
```
Comment (asyncLiz) [inline demos/top-app-bar_iframe/index.html:31]: nit: any reason to combine these two lines together?

File: demos/index.html [hunk 4]
```diff
@@ -20,15 +20,16 @@
...
   <script type="module" src="./index.js"></script>
   <script nomodule src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
   <script nomodule src="index.es5.js"></script>
   <style>
     .unresolved main {
       opacity: 0;
     }
-  </style>
-  <link href="./shared/shared.css" rel="stylesheet">
+  </style><link href="../shared/shared.css" rel="stylesheet">
```
Comment (asyncLiz) [inline demos/index.html:32]: another line combo nit

File: demos/dialog/index.js (hunk 1 above)
Comment (taylor-vann) [reply demos/dialog/index.js]: :|  it is, another branch. will remove.

File: demos/dialog/index.html (hunk 2 above)
Comment (taylor-vann) [reply demos/dialog/index.html]: :|  :|

File: demos/index.html (hunk 4 above)
Comment (taylor-vann) [reply demos/index.html:32]: interesting. two for two. new line.

File: demos/top-app-bar_iframe/index.html (hunk 3 above)
Comment (taylor-vann) [reply demos/top-app-bar_iframe/index.html:31]: no, lets make them two lines.

Comment (taylor-vann) [reply demos/top-app-bar_iframe/index.html:31]: hmmm. i'll keep an i on batch find/replace. definitely wouldn't do this on purpose lol

File: demos/index.html [hunk 5]
```diff
@@ -28,7 +30,7 @@
       opacity: 0;
     }
   </style>
-  <link href="./shared/shared.css" rel="stylesheet">
+  <link href="../shared/shared.css" rel="stylesheet">
```
Comment (asyncLiz) [inline demos/index.html:33]: Is this intentional?

Comment (taylor-vann) [reply demos/index.html:33]: good catch. it is not intended!

**Pull request merged**